src/shared/UI/EmptyCatalog/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from "react-i18next";
+
+export default function EmptyCatalog() {
+  const { t } = useTranslation();
+
+  const messages = t("EmptyCatalogMsgs", { returnObjects: true });
+
+  return (
+    <section className="bg-white text-secondary-900 w-full  h-full py-[118px]  font-rubik rounded-xl max-lg:ml-4 max-lg:mr-4 ">
```

Comment:
Please, use try to use standard numbers for padding (https://tailwindcss.com/docs/padding) for example py-28 or py-32

Reply:
done